src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -83,5 +83,13 @@ public function process(ContainerBuilder $container)
         if (class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
             $container->getDefinition('twig.extension.expression')->addTag('twig.extension');
         }
+
+        if (class_exists(Intl::class)) {
+            $container->getDefinition('twig.extension.intl_bridge')->addTag('twig.extension');
+        }
+
+        if ($container->hasParameter('locale')) {
+            $container->getDefinition('twig.extension.intl_bridge')->replaceArgument(0, $container->getParameter('locale'));
```

Comment:
getting the parameter in the pass is wrong. When injecting a parameter, you should use ``%locale%`` so that it is resolved as late as possible.